packages/synapse-react-client/src/components/OrientationBanner/OrientationBanner.tsx

```diff
@@ -83,10 +86,7 @@ function OrientationBanner(props: OrientationBannerProps) {
         }
         setShowBanner(false)
       }}
-      sx={{
-        ...defaultSx,
-        ...sx,
-      }}
+      sx={spreadSx(defaultSx, sx)}
```

Comment:
I learned that you can pass an array of SxProps to `sx` and MUI will handle merging them, but we need to use a new utility to do so.